kernel/Base/MeshManipulator_Impl.h

```diff
@@ -1119,8 +1119,8 @@ void MeshManipulator<DIM>::addEdgeBasisFunctionSet(
 
 template <std::size_t DIM>
 Base::Element *MeshManipulator<DIM>::addElement(
-    const std::vector<std::size_t> &globalNodeIndexes, std::size_t owner,
-    bool owning) {
+    const std::vector<std::size_t> &globalNodeIndexes, Zone *zone,
```

Comment:
Could we use a reference here or can it be NULL?

Reply:
It will (unfortunately) be null in the reader. I would like it differently, but that would require quite some restructuring.